specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/virtualNetwork.json

```diff
@@ -1496,11 +1500,20 @@
         },
         "privateEndpointNetworkPolicies": {
           "type": "string",
+          "default": "Enabled",
           "description": "Enable or Disable apply network policies on private end point in the subnet."
         },
         "privateLinkServiceNetworkPolicies": {
           "type": "string",
+          "default": "Enabled",
           "description": "Enable or Disable apply network policies on private link service in the subnet."
```

Comment:
You should write it in enum format like this:

https://github.com/Azure/azure-rest-api-specs/blob/5015df257c6f0f8f55e2491a5c13b6ac589c5288/specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/applicationGateway.json#L1786-L1797